temporal-sdk/src/test/java/io/temporal/activity/TestLocalActivities.java

```diff
@@ -0,0 +1,46 @@
+package io.temporal.activity;
+
+import java.time.Duration;
+import java.util.Hashtable;
+import java.util.Map;
+
+public class TestLocalActivities {
```

Comment:
Might be better not to have this class at all. Why not just define an interface and separate class?

Reply:
Good point!